packages/react/src/providers/FuelProvider.tsx

```diff
@@ -24,6 +25,7 @@ export function FuelProvider({
       <FuelHooksProvider fuelConfig={fuelConfig}>
         <FuelUIProvider theme={theme} fuelConfig={fuelConfig}>
           <Connect />
+          <WebWallet />
```

Comment:
I think this should be optional, added by the DApp in a opt-in manner.

Reply:
It makes total sense. I have updated the PR